app/ui/src/main/res/menu/accounts_option.xml

```diff
@@ -5,19 +5,19 @@
     <item
         android:id="@+id/search"
         android:icon="?attr/iconActionSearch"
-        app:showAsAction="always"
+        app:showAsAction="ifRoom"
```

Comment:
We deliberately use `always`. Please revert in this and all other menu resource files.